db/seeds.rb

```diff
@@ -1,7 +1,13 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+data = CoinInfoService.new.get_info
+
+for item in data 
```

Comment:
The developer uses the 'data' variable in the for loop, identically to how it is done in the python programming language, so, because of that, variable 'data' is list.